python/pyfunc-server/pyfuncserver/protocol/rest/handler.py

```diff
@@ -69,3 +66,27 @@ def post(self, name: str):
     def write_error(self, status_code: int, **kwargs: Any) -> None:
         logging.error(self._reason)
         self.write({"status_code": status_code, "reason": self._reason})
+
+
+class LivenessHandler(tornado.web.RequestHandler):  # pylint:disable=too-few-public-methods
```

Comment:
why not using the existing from kfserver?

Reply:
https://github.com/gojek/merlin/pull/276#discussion_r961250290